suite-native/module-onboarding/src/screens/TrackBalancesScreen.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    OnboardingStackParamList,
+    OnboardingStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Box } from '@suite-native/atoms';
+
+import { GetTrezor } from '../components/GetTrezor';
+import { OnboardingScreen } from '../components/OnboardingScreen';
+import { GetTrezorButton } from '../components/GetTrezorButton';
+import { GraphSvg } from '../components/GraphSvg';
+
+export const TrackBalancesScreen = () => {
+    const navigation =
+        useNavigation<
+            StackNavigationProps<OnboardingStackParamList, OnboardingStackRoutes.TrackBalances>
+        >();
+
+    const handleRedirect = () => {
+        navigation.navigate(OnboardingStackRoutes.ReceiveCoins);
+    };
+
+    return (
+        <OnboardingScreen
+            title="Track balances"
+            subtitle="Easily sync your coin addresses and keep up with the crypto on your hardware wallet without exposing your private data."
+            activeStep={1}
+        >
+            <Box alignSelf="center" flex={2} justifyContent="center">
+                <GraphSvg />
+            </Box>
+            <GetTrezor>
+                <GetTrezorButton redirectTarget={handleRedirect} />
+            </GetTrezor>
+        </OnboardingScreen>
+    );
```

Comment:
Such a `Box` with this style is already in OnboardingScreen. Please remove it here.